tests/conftest.py

```diff
@@ -123,6 +123,22 @@ def url_all_origins(request):
     return request.param
 
 
+# Reads the given number of messages from the websocket, and returns them
```

Comment:
Use a docstring inside the `def` method. This integrates with `pytest --fixtures` and with python docs that are surfaced in IDEs.

Reply:
done